resources/bulkupload/cvsFile/registration_center1.csv

```diff
@@ -1,51 +1,51 @@
-id,name,centerTypeCode,addressLine1,addressLine2,addressLine3,latitude,longitude,locationCode,contactPhone,contactPerson,numberOfKiosks,workingHours,perKioskProcessTime,centerStartTime,centerEndTime,lunchStartTime,lunchEndTime,timeZone,holidayLocationCode,zoneCode,langCode,isActive

```

Comment:
we don't need this file @yashmsonkusare . Please remove this.

Reply:
Ok